### ISSUE TITLE
Invalid version: '8.3.*'

### ISSUE THREAD

**Issue by MetehanYasar11** [Deci-AI/super-gradients#991]
`import torch
from super_gradients.training import models

device = torch.device("cuda:0") if torch.cuda.is_available() else torch.device("cpu")
 
model = models.get("yolo_nas_s", pretrained_weights="coco").to(device)

	
out = model.predict("insan1.mp4", conf=0.6)

out.show()`

after run this, i get this in "env_sanity_check.py" ;
Exception has occurred: InvalidVersion
Invalid version: '8.3.*'
  File "/home/metehan/super-gradients/src/super_gradients/sanity_check/env_sanity_check.py", line 99, in check_packages
    req_version = parse_version(req_version_str)
  File "/home/metehan/super-gradients/src/super_gradients/sanity_check/env_sanity_check.py", line 116, in env_sanity_check
    check_packages()
  File "/home/metehan/super-gradients/src/super_gradients/__init__.py", line 25, in <module>
    env_sanity_check()
  File "/home/metehan/Desktop/yoloNAS/infer_deneme.py", line 2, in <module>
    from super_gradients.training import models
pkg_resources.extern.packaging.version.InvalidVersion: Invalid version: '8.3.*'

why is this happening?

**Comment by BloodAxe** [comment]
Please follow the issue template when opening a new issue. It helps to remove unnecessary back-and-forth for additional information to help you with the issue you're facing.

```
### Describe the bug
A clear and concise description of what the bug is.

### To Reproduce
Steps to reproduce the behavior:
1. Train recipe '...'
2. Change param '...'
3. See error

### Expected behavior
A clear and concise description of what you expected to happen.

### Screenshots
If applicable, add screenshots to help explain your problem.

### Environment:
 - OS [e.g. uname -s -r -m]
 - Relevant HW info, GPU + CUDA [e.g. nvidia-smi]
 - Super Gradients version
 - Python environment [e.g. pip freeze]

### Additional context
Add any other context about the problem here.

```

**Comment by ofrimasad** [comment]
@MetehanYasar11 have you made any changes to `requirements.txt`?
it looks like one of your `requirements.txt` contains version `8.3.` which is illegal (use `8.3` instead)

**Comment by rohith033** [comment]
@MetehanYasar11 i would recommend going through this [yolo-nas-training](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/qat_ptq_yolo_nas.md) and set up accordingly

**Comment by Louis-Dupont** [comment]
@MetehanYasar11 try to install from master branch:  `pip install git+https://github.com/Deci-AI/super-gradients`

**Comment by Louis-Dupont** [comment]
This was fixed in `3.1.2`
